cmd/storage-rest-client.go

```diff
@@ -206,7 +206,8 @@ func (client *storageRESTClient) Healing() *healingTracker {
 	return val.(*healingTracker)
 }
 
-func (client *storageRESTClient) NSScanner(ctx context.Context, cache dataUsageCache) (dataUsageCache, error) {
+func (client *storageRESTClient) NSScanner(ctx context.Context, cache dataUsageCache, updates chan<- dataUsageEntry) (dataUsageCache, error) {
```

Comment:
Does the Internode API version have to change here? @klauspost

Reply:
Yes, if we don't otherwise bump it.